docs/source/getting-started/installation.rst

```diff
@@ -97,6 +97,23 @@ Azure
 
 Hint: run ``az account subscription list`` to get a list of subscription IDs under your account.
 
+IBM
+~~~~~~~~~
+
+To use IBM's services store the following fields in `~/.ibm/credentials.yaml`:
```

Comment:
```suggestion
To use IBM, store the following fields in ``~/.ibm/credentials.yaml``:
```

Reply:
Updated to: `To access IBM's services, store the following fields in ``~/.ibm/credentials.yaml``:`